qiskit/circuit/quantumcircuit.py

```diff
@@ -2110,7 +2110,7 @@ def draw(
 
         Args:
             output: Select the output method to use for drawing the circuit.
-                Valid choices are ``text``, ``mpl``, ``latex``, ``latex_source``.
```

Comment:
The list should not be extended. In order plugin hooks, there are a function to list them (for example `qiskit.transpiler.preset_passmanagers.plugin.list_stage_plugins`). You can refer to this function here.